src/bios_handler.cpp

```diff
@@ -283,7 +285,9 @@ void IbmBiosHandler::saveAmmToVpd(const std::string& i_memoryMirrorMode)
         (i_memoryMirrorMode == "Enabled" ? constants::AMM_ENABLED_IN_VPD
                                          : constants::AMM_DISABLED_IN_VPD)};
 
-    // TODO: Call write keyword API to update the value in VPD.
+    m_manager->updateKeyword(
```

Comment:
The API returns "int". If you are not capturing it, how do you ensure that the execution was successful or not?

Reply:
I shall add a return value check. Should we have a `logging::logMessage()` here if `updateKeyword()` fails?